BUILD-sbi

```diff
@@ -0,0 +1,20 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Is this building the sbi-rs submodule?

Reply:
it's a copy. I have actually added the build file to sbi yet.

Reply:
Since bazel support has been added to `sbi-rs` this should no longer be necessary, right?